packages/web-react/src/components/Grid/useGridSpanStyleProps.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint no-console: ["error", { allow: ["warn"] }] */
+import { ElementType } from 'react';
+import classNames from 'classnames';
+import { SpiritBreakpoints, SpiritGridSpanProps } from '../../types';
+import { useClassNamePrefix } from '../../hooks/useClassNamePrefix';
+
+export interface GridSpanStyles<T> {
+  /** className props */
+  classProps: string;
+  /** Props for the grid element. */
+  props: T;
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore TS7053: Element implicitly has an 'any' type because expression of type 'any' can't be used to index type 'typeof SpiritBreakpoints'.
+const gridBreakpoints = Object.keys(SpiritBreakpoints).map((breakpoint) => SpiritBreakpoints[breakpoint as unknown]);
+
+const getGridSpanClasses = (
+  className: string,
+  props: SpiritGridSpanProps<ElementType>,
+  breakpoints: string[] = gridBreakpoints,
+) => {
+  const gridSpanClasses: string[] = [];
+
+  breakpoints.forEach((breakpoint) => {
```

Comment:
This block of code is the same as in the `useGridStyleProps` hook. The only difference is the `--over` and `--span` modificator. Can we make a utility function for this in a separate file and only use the call to this function, please?

Reply:
created hook `useGridClasses` and called it in functions you mentioned.